<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="3" passed="1">
  <reporter-output>
    <line>
      <![CDATA[Reports and Logs Before Method]]>
    </line>
    <line>
      <![CDATA[Reports and Logs Before Method]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="166" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="166" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
      <class name="testclasses.TestNGReportsAndLogs">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="beforeClass" is-config="true" duration-ms="10" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
          <reporter-output>
            <line>
              <![CDATA[Reports and Logs Before Method]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testTwo()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="testTwo" duration-ms="9" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:968)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at testclasses.TestNGReportsAndLogs.testTwo(TestNGReportsAndLogs.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTwo -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="afterMethod" is-config="true" duration-ms="5" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
          <reporter-output>
            <line>
              <![CDATA[Reports and Logs Before Method]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="testOne()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="testOne" duration-ms="2" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOne -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="afterMethod" is-config="true" duration-ms="27" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="testThree()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="testThree" duration-ms="1" started-at="2020-07-27T10:38:38 EDT" depends-on-methods="testclasses.TestNGReportsAndLogs.testTwo" finished-at="2020-07-27T10:38:38 EDT">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestNGReportsAndLogs.testThree()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestNGReportsAndLogs.testThree()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testThree -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:testclasses.TestNGReportsAndLogs@78e67e0a]" name="afterClass" is-config="true" duration-ms="0" started-at="2020-07-27T10:38:38 EDT" finished-at="2020-07-27T10:38:38 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- testclasses.TestNGReportsAndLogs -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
